src/test/java/com/develhope/spring/deals/controllers/OrderIntegrationTest.java

```diff
@@ -109,4 +112,30 @@ void OrderUpdateTest() throws Exception {
                                 """
                 )).andReturn();
     }
+
+    @Test
+    void createAndDeleteOrder_successfulTest() throws Exception {
+        insertAdmin();
+        insertVehicle();
+
+        this.mockMvc.perform(post("/v1/orders").contentType(MediaType.APPLICATION_JSON).content("""
```

Comment:
Volendo c'è insertOrder() che puoi usare al posto di questa perform, anche perché non è stato inserito il controllo sicurezza in questa mock, quindi forse è qui che trovi l'incastro